code/webauthn/contracts/scripts/deploy.ts

```diff
@@ -1,26 +1,23 @@
-import { utils, Wallet, Provider } from 'zksync-ethers';
-import type { HardhatRuntimeEnvironment } from 'hardhat/types';
-// load env file
-import dotenv from 'dotenv';
-import { ethers } from 'ethers';
-dotenv.config();
-
-const DEPLOYER_PRIVATE_KEY = process.env.WALLET_PRIVATE_KEY || '';
-
-export default async function (hre: HardhatRuntimeEnvironment) {
-  // @ts-expect-error target config file which can be testnet or local
-  const provider = new Provider(hre.network.config.url);
-  const wallet = new Wallet(DEPLOYER_PRIVATE_KEY, provider);
+import { deployer, ethers } from 'hardhat';
```

Comment:
similar to previous with this `deployer` class. Does it even work? I'd rather use a contractFactory